service_container/debug.rst

```diff
@@ -31,3 +31,13 @@ its id:
 .. code-block:: terminal
 
     $ php bin/console debug:container app.mailer
+
+By default, the arguments of the services are not included in the output. Add the
+``--show-arguments`` option to show them:
+
+.. versionadded:: 3.3
```

Comment:
Having the `versionadded` directive between the previous paragraph and the output reads a bit weird.